# HSV PARAM CONFIGURATIONS

# hsv_h_min: 25
# hsv_h_max: 44
# hsv_s_min: 65
# hsv_s_max: 160
# hsv_v_min: 70
# hsv_v_max: 255
hsv_h_min: 33
hsv_h_max: 39
hsv_s_min: 81
hsv_s_max: 152
hsv_v_min: 137
hsv_v_max: 242


# HLS PARAM CONFIGURATION
hls_h_min: 0  
hls_h_max: 82
hls_l_min: 108
hls_l_max: 232
hls_s_min: 0
hls_s_max: 255

# CAMERA PARAMETERS
image_encoding: "rgb8"
use_hsv: true